src/parsers/cdm_parser.jl

```diff
@@ -317,23 +321,24 @@ function gen_csv_parser(gen_raw::DataFrames.DataFrame, Buses::Dict{Int64,Any},co
             fuel_cost = gen_raw[gen,colnames["Fuel Price \$/MMBTU"]]./1000
 
             var_cost = [(_get_value_or_nothing(gen_raw[gen,cn[1]]), _get_value_or_nothing(gen_raw[gen,cn[2]])) for cn in cost_colnames]
-            var_cost = [(c[1],c[1]*c[2]*fuel_cost).*pmax for c in var_cost if !in(nothing,c)]
-            var_cost[2:end] = [(var_cost[i][1],var_cost[i-1][2]+var_cost[i][2]) for i in 2:length(var_cost)]
+            var_cost = [(c[1]*c[2]*fuel_cost, c[2]/baseMVA).*pmax for c in var_cost if !in(nothing,c)]
+            var_cost[2:end] = [(var_cost[i-1][1]+var_cost[i][1], var_cost[i][2]) for i in 2:length(var_cost)]
 
             bus_id =[Buses[i] for i in keys(Buses) if Buses[i]["number"] == gen_raw[gen,colnames["Bus ID"]]]
+
             Generators_dict["Thermal"][gen_raw[gen,colnames["GEN UID"]]] = Dict{String,Any}("name" => gen_raw[gen,colnames["GEN UID"]],
                                             "available" => true,
                                             "bus" => make_bus(bus_id[1]),
-                                            "tech" => Dict{String,Any}("activepower" => gen_raw[gen,colnames["MW Inj"]],
-                                                                        "activepowerlimits" => (min=_get_value_or_nothing(gen_raw[gen,colnames["PMin MW"]]),max=pmax),
-                                                                        "reactivepower" => _get_value_or_nothing(gen_raw[gen,colnames["MVAR Inj"]]),
-                                                                        "reactivepowerlimits" => (min=_get_value_or_nothing(gen_raw[gen,colnames["QMin MVAR"]]),max=_get_value_or_nothing(gen_raw[gen,colnames["QMax MVAR"]])),
-                                                                        "ramplimits" => (up=gen_raw[gen,colnames["Ramp Rate MW/Min"]],down=gen_raw[gen,colnames["Ramp Rate MW/Min"]]),
-                                                                        "timelimits" => (up=gen_raw[gen,colnames["Min Up Time Hr"]],down=gen_raw[gen,colnames["Min Down Time Hr"]])),
-                                            "econ" => Dict{String,Any}("capacity" => pmax,
+                                            "tech" => Dict{String,Any}("activepower" => gen_raw[gen,colnames["MW Inj"]]/baseMVA,
```

Comment:
Maybe this is a dumb question, but can you apply <divide by baseMVA> to the specific columns in the dataframe in one shot?

Reply:
Indeed we could. The thought had crossed my mind, but I thought this might be more obvious, and there probably isn't a real performance penalty. But if you prefer, I'll do what you suggest.

Reply:
Performance should be the same either way.  I would look at it from a code maintenance perspective:  if you ever have to change the calculation again, what's easier and less error-prone?